web/tests/pm_list_data.test.js

```diff
@@ -172,6 +178,7 @@ test("get_conversations bot", ({override}) => {
             status_emoji_info: undefined,
             is_group: true,
             is_bot: false,
+            dms_contain_unread_mention: false,
```

Comment:
Can you extend the test setup such that one of these cases has this value `true`?